ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json

```diff
@@ -1,42 +1,41 @@
 {
     "version": "v1.0",
     "author": "ModelArts Service of Huawei Cloud",
-    "authorEmail": "tianchudong@huawei.com",
+    "authorEmail": "lipengfei44@huawei.com, tianchudong@huawei.com, chenyi61@huawei.com",
     "framework": "MXNet v1.2.1",
-    "codeURL": "https://github.com/huawei-clouds/modelarts-example/blob/eb04ab2830814a70667210ee97c04ddb2aaa0059/MXNet-ImageNet-10min/16_32_dysizelr.py",
```

Comment:
@TccccD sorry I noticed you deleted the `codeURL`. Was that intended?

Reply:
Oh, ok...... We will update later